app/views/posts/show.html.erb

```diff
@@ -1,7 +1,21 @@
-<p style="color: green"><%= notice %></p>
+<p style="color: green"><%= flash[:success] %></p>
 
 <%= render @post %>
```

Comment:
- [ ] @AbigiyaTY great work on your app am impressed, however, you forgot to add a button to like a post, like you did for the comment, this is a project requirement 
![image](https://user-images.githubusercontent.com/61408860/208226325-0f9c4b91-4338-464d-a73c-6924bfacbb0b.png)
 
![image](https://user-images.githubusercontent.com/61408860/208226300-3477e762-462e-409a-8b3d-270153d28d09.png)

Reply:
Hi @danmainah , Thank for your review

The button is to increment the like.
Here is the proof of that

Before the button click
![image](https://user-images.githubusercontent.com/98043302/208236293-db9186aa-aeb2-4e50-9970-4f6e4d8efaac.png)

After the button click
![image](https://user-images.githubusercontent.com/98043302/208236417-da9df9a4-5c09-4e06-b4e1-7eab40667cb8.png)

Reply:
This requirement has been met ✔️